index.html

```diff
@@ -29,31 +29,185 @@
+        </div>
+      </div>
+
+      <!-- multipurpose section1 -->
+      <div class="multipurpose-section1">
+        <div class="image"></div>
+
+        <h2>Tonic</h2>
+        <span class="canopy">CANOPY</span>
+        <span class="dev">• Back End Dev • 2015</span>
+        <div class="daily-sign">
+          A daily selection of privately<br />
+          personalized reads; no accounts or sign-ups required.
+        </div>
+
+        <button class="html">html</button>
+        <button class="css">css</button>
+        <button class="javascript">javascript</button>
+
+        <div>
+          <button class="see-project">See Project</button>
+        </div>
+      </div>
+
+      <!-- multipost section -->
+      <div class="post-section">
+        <div class="image"></div>
+
+        <h2>Tonic</h2>
+        <span class="canopy">CANOPY</span>
+        <span class="dev">• Back End Dev • 2015</span>
+        <div class="daily-sign">
+          A daily selection of privately<br />
+          personalized reads; no accounts or sign-ups required.
+        </div>
+
+        <button class="html">html</button>
+        <button class="css">css</button>
+        <button class="javascript">javascript</button>
+
+        <div>
+          <button class="see-project">See Project</button>
+        </div>
+      </div>
+      <!-- stories section -->
+      <div class="stories-section">
+        <div class="image"></div>
+        <h2>Tonic</h2>
+        <span class="canopy">CANOPY</span>
+        <span class="dev">• Back End Dev • 2015</span>
+        <div class="daily-sign">
+          A daily selection of privately<br />
+          personalized reads; no accounts or sign-ups required.
+        </div>
+
+        <button class="html">html</button>
+        <button class="css">css</button>
+        <button class="javascript">javascript</button>
+
+        <div>
+          <button class="see-project">See Project</button>
+        </div>
       </div>
 
-      <div class="connect">
-        <p>LET’S CONNECT</p>
+      <div class="about-section">
+        <h2>
+          About<br />
+          Myself
+        </h2>
+        <p>
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <p class="connect">LET’S CONNECT</p>
+
+        <div class="social-icons">
+          <span class="iconify" data-icon="iconoir:github"></span>
+          <span class="iconify" data-icon="eva:linkedin-fill"></span>
+          <span class="iconify" data-icon="fontisto:angelist"></span>
+          <span class="iconify" data-icon="ei:sc-twitter"></span>
+          <span class="iconify" data-icon="bx:bxl-medium"></span>
+        </div>
+
+        <div>
+          <p class="resume">Get my resume</p>
+        </div>
```

Comment:
- [ ] The button ("Get my resume") should be a link `a` or `button` tag, and follow the button styling: Border, Border color, Border radius, Active, Hover and Pressed (interactions found in the Figma design).